client/src/css/overview.css

```diff
@@ -35,20 +35,38 @@
   max-width: 100px;
   display: flex;
   flex-flow: column nowrap;
-  align-items: flex-start;
+  align-items: center;
+}
+
+.gallery-thumbnails-viewport {
+  overflow: hidden;
+}
+
+.gallery-thumbnails-all {
+  transform: translate(0, 0);
+  margin: 10px;
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  row-gap: 20px;
+  transition: all 400ms ease;
 }
 
 .main-image-pane {
-  background-color: #dddddd;
+  /* background-color: #dddddd; */
```

Comment:
This is now fixed. Previously I had a div that contained the thumbnails and the main image, and that div was part of the top-level Overview grid. I got rid of that div and revised the grid to include the thumbnails and main image (so now it has three columns instead of two). Then I had to position the main image absolutely relative to its container. It was painful.